libfwupdplugin/fu-udev-device.c

```diff
@@ -517,6 +517,12 @@ fu_udev_device_probe(FuDevice *device, GError **error)
 	fu_device_add_instance_str(device, "DRIVER", priv->driver);
 	fu_device_build_instance_id_quirk(device, NULL, subsystem, "DRIVER", NULL);
 
+	/* add the modalias */
+	fu_device_add_instance_strsafe(device,
+				       "MODALIAS",
+				       g_udev_device_get_property(priv->udev_device, "MODALIAS"));
+	fu_device_build_instance_id(device, NULL, subsystem, "MODALIAS", NULL);
```

Comment:
I use fu_device_build_instance_id_quirk().
If I use fu_device_build_instance_id() here, it always fails to compile in my chromeOS development environment.

Reply:
Fixed.
Is the quirk file correct?

Reply:
> Fixed. Is the quirk file correct?

Thanks, that's right.